### PR TITLE
Translation

### PR DESCRIPTION
**Pull request by radcapricorn**

### CODE REVIEW

File: vibed/basics-asynchronous-i-o.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Основы и асинхронный ввод/вывод
+
+При использовании параметров сборки по умолчанию
```
Comment (dmi7ry) [inline vibed/basics-asynchronous-i-o.md]: запятая после «умолчанию»

File: vibed/basics-asynchronous-i-o.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+текущее волокно исполняет функцию `yield`, оставляя
+тем самым возможность исполнения других операций.
+Когда данные наконец поступят, мы просто возобновляем
+исполнение:
+
+    // Возможна блокировка, но она происходит
+    // незаметно. Если сокет готов, vibe.d
+    // вернёт результат.
+    line = connection.readLine();
+    // Так же возможна блокировка
```
Comment (dmi7ry) [inline vibed/basics-asynchronous-i-o.md]: «также» слитно

**Comment by radcapricorn** [comment]
"Root package dlang-tour references unknown package dyaml" o_O

**Comment by dmi7ry** [comment]
@wilzbach LDC test failed. Heeeelp.

**Comment by wilzbach** [comment]
>  LDC test failed. Heeeelp.

Seemed to be a transient problem. Simply restarting the build worked :)

**Pull request merged**